tests/examples/whitelist-css.css

```
@@ -0,0 +1,15 @@
+.paragraph {
+	height: 100%;
+}
+
+.icon {
+	height: 100%;
+}
+
+.icon-arrow {
+	width: 10px;
+}
+
+.icon-search {
```

I don't see how this is different from the test of `.icon-arrow` but what could make this line interesting is...:
```suggestion
.icon-search, .not-icon-something {
```